components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,390 @@
-                        maxLength={field.maxLength}
+                <Flex flexDirection="column" my={4} mx={1} width={[320, 450]} color="black.800" letterSpacing="0.2px">
+                  <StyledCheckbox
+                    name="authorization"
+                    required
+                    fontSize="12px"
+                    label={
+                      <FormattedMessage
+                        id="createorganization.authorization.label"
+                        defaultMessage="I verify that I am an authorized representative of this organization and 
+                            have the right to act on its behalf."
                       />
-                    ),
-                )}
-              </div>
-            ))}
+                    }
+                    onChange={({ checked }) => {
+                      setAuthorization(checked);
+                    }}
+                  />
+                  <Flex justifyContent={['center', 'left']} my={4}>
+                    <StyledButton
+                      fontSize="13px"
+                      minWidth="148px"
+                      minHeight="36px"
+                      buttonStyle="primary"
+                      type="submit"
+                      loading={loading}
+                      onSubmit={handleSubmit}
+                      data-cy="cof-form-submit"
+                    >
+                      <FormattedMessage id="create.org.title" defaultMessage="Create Organization" />
+                    </StyledButton>
+                  </Flex>
 
-            <div className="tos">
-              <label>{intl.formatMessage(this.messages['tos.label'])}</label>
-              <div data-cy="tos">
-                <StyledCheckbox
-                  name="tos"
-                  id="oc-tos-checkbox"
-                  required
-                  onChange={({ target }) => this.handleChange('tos', target.checked)}
-                  label={
+                  <Box
+                    textAlign="left"
+                    fontSize="12px"
+                    minHeight="24px"
+                    color="black.500"
+                    fontWeight="normal"
+                    letterSpacing="0.5px"
+                  >
                     <FormattedMessage
-                      id="createcollective.tos.label"
-                      defaultMessage="I agree with the {toslink} of Open Collective."
-                      values={{
-                        toslink: (
-                          <StyledLink href="/tos" openInNewTab>
-                            <FormattedMessage id="tos" defaultMessage="terms of service" />
-                          </StyledLink>
-                        ),
-                      }}
+                      id="createOrganization.tos"
+                      defaultMessage="By joining, you agree to our <strong><I18nTOSLink>Terms of Service</I18nTOSLink></strong> and <strong><I18nPrivacyLink>Privacy Policy</I18nPrivacyLink></strong>
```

Comment:
This `FormattedMessage` is displaying too many things. Can we re-use the existing messages here? Copy pasting the code from https://github.com/opencollective/opencollective-frontend/blob/feat/edit-collective-sections-new-format/components/contribution-flow/StepProfileGuestForm.js#L138 should do the job

Reply:
Done